### DISCUSSION TITLE
Confluent 6.X

### DISCUSSION THREAD

**Discussion by vanwagenenj** [AbsaOSS/ABRiS#193] [General]
Hello,

Are there any plans to update this project so it works with Confluent 6.X?

I'm trying to get this to work with the latest dependencies and it seems that there are methods this uses that are no longer available in the latest versions of things like spark-avro.

Best Regards,
John Van Wagenen

**Comment by cerveada** [comment]
We want to stay compatible with spark and there are some compatibility issues that keep us lock in version 5.3 (more here: #165)

We believe the compatibility issue can be resolved with Avro 1.10 which will be supported in Spark 3.2.0 (see https://issues.apache.org/jira/browse/SPARK-27733)

The Only way this could be solved right now is to add support for both versions via reflection, but that might be complicated.

**Reply by vanwagenenj** [reply]
Thank you for your reply and for this information!
